### PR TITLE
introduce ActionGroup.dropdown_width property

### PR DESCRIPTION
**Pull request by denys-duchier**
This patch adds the dropdown_width property to ActionGroup and makes it easy to accommodate a dropdown list with wide items.  Full disclosure: it is already possible (but imho much less intuitive) to do something similar using the minimum_width property on the LAST item.

### CODE REVIEW

File: kivy/uix/actionbar.py [hunk 1]
```diff
@@ -291,7 +295,7 @@ def _toggle_dropdown(self, *largs):
         children = ddn.container.children
 
         if children:
-            ddn.width = max([self.width, children[0].minimum_width])
+            ddn.width = self.dropdown_width or max([self.width, children[0].minimum_width])
```
Comment (FeralBytes) [inline kivy/uix/actionbar.py]: This always assigns what ever self.dropdown_width, which is the left side of the or statement. But if I don't set that, then my width will always be 0.

File: kivy/uix/actionbar.py [hunk 2]
```diff
@@ -249,6 +249,10 @@ class ActionGroup(ActionItem, Spinner):
        defaults to 'normal'.
     '''
 
+    dropdown_width = NumericProperty(0)
+    '''If non zero, provides the width for the associated DropDown.
```
Comment (FeralBytes) [inline kivy/uix/actionbar.py]: In my opinion, your documentation is too minimal, especially for a new property.
Should be more like:
```python
'''If non zero, provides the width for the associated DropDown.
.. versionadded:: 1.9.0
:attr:`dropdown_width` is an :class:`~kivy.properties.NumericProperty` and defaults to 0.
'''
```

File: kivy/uix/actionbar.py (hunk 1 above)
Comment (denys-duchier) [inline kivy/uix/actionbar.py]: it should definitely prefer dropdown_width. that was the whole point.

File: kivy/uix/actionbar.py (hunk 2 above)
Comment (denys-duchier) [inline kivy/uix/actionbar.py]: I agree.  I even wanted to add a test and that's when I discovered that the graphics tests are messed up https://github.com/kivy/kivy/issues/3130

Comment (kived) [inline kivy/uix/actionbar.py]: `versionadded` should come after the property/default descriptions, but both changes are required before this PR can be merged.

**Comment by KeyWeeUsr** [comment]
@denys-duchier Please resolve the conflict in actionbar.py and add the documentation properly (order: description, property, version). Seems to be good for merge. ^^

**Comment by dessant** [comment]
Same as https://github.com/kivy/kivy/pull/3625#issuecomment-264732811.

**Pull request merged**